code/modules/mob/living/simple_animal/bot/floorbot.dm

```diff
@@ -145,6 +145,7 @@
 	..()
 	if(emagged == 2)
 		if(user)
+			balloon_alert(user, "opposite day uploaded")
 			to_chat(user, "<span class='danger'>[src] buzzes and beeps.</span>")
```

Comment:
Don't need this anymore.

Reply:
In the future we should replace this with a low radius, chatless Runechat emote.